lib/client.js

```diff
@@ -749,6 +752,22 @@ MqttClient.prototype._cleanUp = function (forced, done) {
   }
 }
 
+/**
+ * _flush - Invoke (if it exists) all of the callbacks in an outgoing queue with an
+ *          error that we've disconnected
+ * @api private
+ */
+MqttClient.prototype._flush = function (queue) {
```

Comment:
There is no need to have this as an instance method, it can be a top-level function.